.travis.yml

```diff
@@ -2,6 +2,4 @@ dist: trusty
 sudo: required
 language: java
 jdk:
-  - oraclejdk7
-  - openjdk7
```

Comment:
what is the reason of removing this? are you sure old soapUI version runs under JDK8 only?